test/src/tests/account.test.js

```diff
@@ -1,41 +1,30 @@
 import {
   vAvatar,
-  vBtn,
-  vListItem,
-  vIcon,
 } from 'jest-puppeteer-vuetify';
-import { registerNewUser } from '../util';
+import {
+  logoutUser, registerNewUser, LOGIN_BUTTON_TEXT, authorize,
+} from '../util';
 
 describe('account management', () => {
   it('logs the user out', async () => {
     await registerNewUser();
 
     await expect(page).toClickXPath(vAvatar('??'));
-    await page.waitFor(500);
-    await expect(page).toClickXPath(vListItem({ content: 'Logout', action: vIcon('mdi-logout') }));
+
+    await logoutUser();
```

Comment:
This was actually a deliberate design decision, not a redundant bit of code.

For frontend tests, I've found that there are frequently multiple ways of getting the GUI into a certain state, some of which are faster than others. The `util` module is a collection of helpers so tests can set themselves up quickly and easily. All the `util` helpers should only be used when you need something done and you don't care how. 

When writing the test, `utils` should only be used to set up and tear down. The actual thing being tested should be written out explicitly, even if it is technically duplicating code that already exists in the helper. We might eventually optimize the helper and do things a different way, in which case the test is now testing something completely different, with no indication that the behavior has changed.

I've found that violating DRY in this case makes maintaining tests more reliable, but I'm happy to discuss.

Reply:
I was wondering about that, thanks for the explanation. Thinking about future tests, some of the operations currently being tested (signing up a user, logging in, registering a dandiset, etc) will be needed to setup more involved tests, so having those basic tests be explicit while we use helper functions for the more involved tests makes sense to me.